spark/1.9/README.md

```diff
@@ -0,0 +1,161 @@
+	Issues: https://docs.mesosphere.com/support/
+```
+
+Note that while the DC/OS CLI subcommand `spark` is immediately available, it takes a few moments for Spark to start running in the cluster.
+
+Let's first check the DC/OS CLI and its new subcommand `spark`:
+
+```bash
+$ dcos spark
+Usage:
+    dcos spark --help
+    dcos spark --info
+    dcos spark --version
+    dcos spark --config-schema
+    dcos spark run --help
+    dcos spark run --submit-args=<spark-args>
+                   [--docker-image=<docker-image>]
+                   [--verbose]
+    dcos spark status <submissionId> [--verbose]
+    dcos spark log <submissionId>
+                   [--follow]
+                   [--lines_count=<lines_count>]
+                   [--file=<file>]
+    dcos spark kill <submissionId> [--verbose]
+    dcos spark webui
+```
+
+Now, we validate if Spark is running and healthy, in the cluster itself. For this, go to the DC/OS UI and you should see Spark there under the `Services` tab:
+
+![Services](img/services.png)
+
+You can get to this page by using the `dcos spark webui` command:
+
+![Spark Drivers page](img/spark-drivers.png)
+
+## Launch a Spark job
+
+Now that you've set up Spark, it's time to launch your first Spark job. We will use one of the [existing examples](https://github.com/apache/spark/blob/master/examples/src/main/scala/org/apache/spark/examples/SparkPi.scala) that comes with Spark.
+
+Note that when you execute the `dcos spark run` command the first time, DC/OS downloads the necessary assets to your local machine and that can take (depending on your Internet connection) some 10 mins or more.
+
+```bash
+$ dcos spark run --submit-args='-Dspark.mesos.coarse=true --driver-cores 1 --driver-memory 1024M --class org.apache.spark.examples.SparkPi https://downloads.mesosphere.com/spark/assets/spark-examples_2.10-1.4.0-SNAPSHOT.jar 30'
+downloads.mesosphere.com/spark/assets/spark-examples_2.10-1.4.0-SNAPSHOT.jar 30'
+Spark distribution spark-2.1.0-1-bin-2.6 not found locally.
+It looks like this is your first time running Spark!
+Downloading https://downloads.mesosphere.com/spark/assets/spark-2.1.0-1-bin-2.6.tgz...
+Extracting spark distribution /Users/yishan/.dcos/spark/dist/spark-2.1.0-1-bin-2.6.tgz...
+Successfully fetched spark distribution https://downloads.mesosphere.com/spark/assets/spark-2.1.0-1-bin-2.6.tgz!
+127.0.0.1 - - [13/Apr/2017 11:21:42] "POST /v1/submissions/create HTTP/1.1" 200 -
+127.0.0.1 - - [13/Apr/2017 11:21:42] "GET /v1/submissions/status/driver-20170413182142-0001 HTTP/1.1" 200 -
+Run job succeeded. Submission id: driver-20170413182142-0001
+```
+
+Note the submission ID in the last line, above, in our example `driver-20170413182142-0001`: this allows to track the job via the CLI as we will see below. When you look at the Spark drivers page you should see the job as well:
+
+![Spark finished job](img/spark-finished-job.png)
+
+Alternatively, you can use the CLI to verify the job status:
```

Comment:
The output you shared below is from when the job has completed, so you should say that, and explain that this will change based on when you run the status command.